src/experiments.js

```diff
@@ -66,6 +66,15 @@ export function isCanary(win) {
   return !!(win.AMP_CONFIG && win.AMP_CONFIG.canary);
 }
 
+/**
+ * Returns binary type, e.g., canary, control, or production.
+ * @param {!Window} win
+ * @return {string}
+ */
+export function getBinaryType(win) {
+  return (win.AMP_CONFIG && win.AMP_CONFIG.type) || 'unknown';
```

Comment:
If I'm reading this correctly, won't the expression within parentheses evaluate to true in the canonical case, and the function always return the `type` corresponding to `1` irrespective of its actual value? Even if this does work, for readability, perhaps change it to the following?

```
return (win.AMP_CONFIG ? win.AMP_CONFIG.type : 'unknown');
```

Reply:
AMP_CONFIG.type is a string, not a numerical value, so I added a test for its existence before returning it directly.